pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -26,6 +26,21 @@ import (
 	"github.com/openshift/origin/pkg/image/importer"
 )
 
+const (
+	// DockerRegistryURLEnvVar is a mandatory environment variable name specifying external url of internal
+	// docker registry. All references to pushed images will be prefixed with its value.
+	DockerRegistryURLEnvVar = "DOCKER_REGISTRY_URL"
+
+	// EnforceQuotaEnvVar is an boolean environment variable that allows to turn quota enforcement on or off.
+	// By default, quota enforcement is off. Recognized values are "true", and "false".
+	EnforceQuotaEnvVar = "OPENSHIFT_ENFORCE_QUOTA"
+
+	// ProjectCacheTTLEnvVar is an environment variable specifying an eviction timeout for project quota
```

Comment:
All of these should be coming from the docker registry config pattern (passed to the middleware) using the same pattern that already exists for env.  We shouldn't be using os.Getenv anywhere in this package.